experimental/framework/fluid-static/src/FluidStatic.ts

```diff
@@ -70,6 +75,12 @@ export class FluidContainer {
         return this.rootDataObject.createDataObject<T>(dataObjectClass, id);
     }
 
+    public getId() {
+        return this.container.clientId;
+    }
+
+    public readonly audience: IAudience;
```

Comment:
Let's keep public variables at the top of the class. Above constructor below private variables.

Reply:
ok, so just functions down below